lib/api/queries/weather.js

```diff
@@ -44,23 +44,19 @@ export const dailyWeather = (bucket) => {
   date.setHours(0, 0, 0, 0);
   return flux`data = from(bucket: "${bucket}")
       |> range(start: ${date}, stop: now())
-      |> filter(fn: (r) => r["_measurement"] == "lsst.sal.ESS.pressure" or r["_measurement"] == "lsst.sal.ESS.dewPoint" or r["_measurement"] == "lsst.sal.ESS.temperature" or r["_measurement"] == "lsst.sal.ESS.relativeHumidity" or r["_measurement"] == "lsst.sal.ESS.airFlow")
-      |> filter(fn: (r) => r["_field"] == "relativeHumidity" or r["_field"] == "dewPoint" or r["_field"] == "temperature0" or r["_field"] == "speed" or r["_field"] == "pressure0" or r["_field"] == "direction")
+      |> filter(fn: (r) => r["_measurement"] == "lsst.sal.ESS.temperature")
```

Comment:
trimmed down some of the queries for speed, temperature is our only per-day widget so we don't need to fetch anything else and it only shows high/low so the mean isn't necessary